app/blueprint-tests/page-object.js

```diff
@@ -0,0 +1 @@
+export { default } from 'ember-cli-page-object/blueprint-tests/page-object';
```

Comment:
Hmm. I think those were added by ember-cli-blueprint-test-helpers. I'll check it out.